src/sentry/workflow_engine/migration_helpers/issue_alert_conditions.py

```diff
@@ -105,3 +106,14 @@ def create_new_high_priority_issue_condition(
         condition_result=True,
         condition_group=dcg,
     )
+
+
+@data_condition_translator_registry.register(LevelCondition.id)
```

Comment:
there's also `LevelFilter` -- you might have to add `enable_reverse_lookup=False` to `data_condition_translator_registry` to allow double registration

Reply:
I was going to add all of the double registrations in another PR

Reply:
it wouldn't be hard to add it here :P